WordPress/Classes/Models/BlogSettings.swift

```diff
@@ -0,0 +1,203 @@
+import Foundation
+
+
+/**
+ *  @class           BlogSettings
+ *  @brief           This class encapsulates all of the settings available for a Blog entity
+ */
+public class BlogSettings : NSManagedObject
+{
+    // MARK: - Relationships
+    
+    /**
+    *  @details Maps to the related Blog.
+    */
+    @NSManaged var blog : Blog?
+    
+
+    
+    // MARK: - General
+    
+    /**
+    *  @details Represents the Blog Name.
+    */
+    @NSManaged var name : String?
+    
+    /**
+     *  @details Stores the Blog's Tagline setting.
+     */
+    @NSManaged var tagline : String?
+    
+    /**
+     *  @details Stores the Blog's Privacy Preferences Settings
+     */
+    @NSManaged var privacy : NSNumber?
```

Comment:
I know we're just moving things around on this PR, but for the next one, this wants to be a transformable enum :)